src/github/github.ts

```diff
@@ -22,7 +22,7 @@ import type {
 
 // TODO: Refactor this whole file into a class for better use in dataview queries, etc
 
-const tokenMatchRegex = /repo:(.+)\//;
+const tokenMatchRegex = /repo:([\w\-]+)\//;
```

Comment:
This line fixes a subtle bug where the regex captured everything until the last (instead of next) slash and crossed word boundaries if it was missing.